chsdi/templates/htmlpopup/zeitreihen.mako

```diff
@@ -1,9 +1,122 @@
 <%inherit file="base.mako"/>
 
+<%!
+import datetime
+import urllib
+import urllib2
+from pyramid.url import route_url
+import chsdi.lib.helpers as h
+import markupsafe
+
+def br(text):
+    return text.replace('\n', markupsafe.Markup('<br />'))
+
+tileUrlBasePath = 'http://historicalmaps.geo.admin.ch/tiles'
+
+def determinePreviewUrl(bvnummer):
+
+    def getPreviewImageUrl(bvnummer):
+        return tileUrlBasePath + '/' + bvnummer + '/quickview.jpg'
+
+    def getZeroTileUrl(bvnummer):
+        return tileUrlBasePath + '/' + bvnummer + '/0/0/0.jpg'
+
+    class HeadRequest(urllib2.Request):
+        def get_method(self):
+            return 'HEAD'
+
+    def testForUrl(url):
+        response = None
```

Comment:
you can check the existence pf a resource with a helper in lib. much easier it returns True or False.